public/projects/installfest.md

```diff
@@ -20,15 +20,13 @@ Before the class starts, please install the following:
 ## Node.js (JavaScript)
 
   * **Install** Node.js from <https://nodejs.org/>
-  * version 10.10.0 is preferred
+  * version 10.X.X LTS is preferred
     * run `node -v` in terminal to check
     * if you can't install v10, anything past 6.0 is probably okay
 
 ## Git
 
-* **Install** git from...
-  * for Mac and Linux: <https://git-scm.com/downloads>
-  * for Windows: <https://gitforwindows.org/>
+* **Install** git from  <https://git-scm.com/downloads>
```

Comment:
Does the package at https://git-scm.com/download/win contain "git bash"? (I'll have to reboot into Windows before I can see for myself.)

Reply:
In your email you mentioned “Bash on Windows” -- are you talking about WSL? (Windows Subsystem for Linux) 

I have heard good things about WSL but I'm still wary of forcing our Windows students to use it. There are many things that work just a *little* bit differently... for instance, when you launch it, you are not in your Windows home directory, and would have to `cd /mnt/c/Users/myname` before doing any work.